examples/GetSetVal/Example1_GetSetPortSettings/Example1_GetSetPortSettings.ino

```diff
@@ -0,0 +1,100 @@
+/*
+  Configuring port settings using the newer getVal/setVal methods
+  By: Nathan Seidle
+  SparkFun Electronics
+  Date: October 23rd, 2020
+  License: MIT. See license file for more information but you can
+  basically do whatever you want with this code.
+
+  This example shows how to query a Ublox module for its UART1 settings and
+  then change them if the settings aren't what we want.
+
+  Note: getVal/setVal/delVal are only support in u-blox protocol versions 27 and higher.
+
+  Feel like supporting open source hardware?
+  Buy a board from SparkFun!
+  ZED-F9P RTK2: https://www.sparkfun.com/products/15136
+  NEO-M8P RTK: https://www.sparkfun.com/products/15005
```

Comment:
getVal and setVal are only supported by the 9 family... NEO and SAM have got to go...
(Or am I just being pedantic?!)

Reply:
Ah! I was mistakenly under the impression that get/setVal were supported under v27+ but I am wrong, you are right. That certainly is an argument for keeping the library as it is (supports *all* modules whereas get/set will only support ZED and a smattering of future others).

Removing from ex. And because of this, I will remove the sub-folder of getsetval. It really is ZED only at this point.